sample-install/snirouting_tls/base/kroxylicious-config.yaml

```
@@ -0,0 +1,38 @@
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kroxylicious-config
+data:
+  config.yaml: |
+    adminHttp:
+      endpoints:
+        prometheus: {}
+    virtualClusters:
+      demo:
```

The kafka resource is named my-cluster, so the a name like `my-cluster-proxy` would help make it clear what's going on.
```suggestion
      my-cluster-proxy:
```